Rules/CommonScripts/ChatCommands/BlobCommands.as

```diff
@@ -0,0 +1,178 @@
+#include "ChatCommand.as"
+#include "MakeSeed.as";
+#include "MakeCrate.as";
+#include "MakeScroll.as"
+
+class PineTreeCommand : ChatCommand
+{
+	PineTreeCommand()
+	{
+		super("pinetree", "Spawn a pine tree seed");
+		AddAlias("pineseed");
+		AddAlias("tree");
+		AddAlias("seed");
+	}
+
+	void Execute(string[] args, CPlayer@ player)
+	{
+		if (!isServer()) return;
+
+		CBlob@ blob = player.getBlob();
+		if (blob !is null)
+		{
+			Vec2f pos = blob.getPosition();
+			server_MakeSeed(pos, "tree_pine", 600, 1, 16);
```

Comment:
Also affected by the blob spawning comment (posting a comment for each of them for tracking)